### PR TITLE
Mandd/user manual fixes

### LINKED ISSUE #1767: [DEFECT] Codes <csv> node explanation
### Thank you for the defect report

- [X] I am using the latest version of `RAVEN`.
- [X] I have read the [Wiki](https://github.com/idaholab/raven/wiki).
- [ ] I have created a [minimum, reproducible example](https://stackoverflow.com/help/minimal-reproducible-example)
that demonstrates the defect.

### Defect Description

The <csv> nodes for code allows the users to decided if they want a csv to be generated for each code run. However, this node is not reported in the user manual.

### Steps to Reproduce

no need to reproduce this error

### Expected Behavior

<csv> node is not indicated in the report

### Screenshots and Input Files

_No response_

### OS

MacOS

### OS Version

Monterey

### Dependency Manager

CONDA

### For Change Control Board: Issue Review

- [X] Is it tagged with a type: defect or task?
- [X] Is it tagged with a priority: critical, normal or minor?
- [X] If it will impact requirements or requirements tests, is it tagged with requirements?
- [X] If it is a defect, can it cause wrong results for users? If so an email needs to be sent to the users.
- [X] Is a rationale provided? (Such as explaining why the improvement is needed or why current code is wrong.)

### For Change Control Board: Issue Closure

- [x] If the issue is a defect, is the defect fixed?
- [x] If the issue is a defect, is the defect tested for in the regression test system?  (If not explain why not.)
- [x] If the issue can impact users, has an email to the users group been written (the email should specify if the defect impacts stable or master)?
- [x] If the issue is a defect, does it impact the latest release branch? If yes, is there any issue tagged with release (create if needed)?
- [x] If the issue is being closed without a pull request, has an explanation of why it is being closed been provided?

### PR DESCRIPTION
**Pull request by mandd**
--------
Pull Request Description
--------
##### What issue does this change request address? (Use "#" before the issue to link it, i.e., #42.)
Closes #1767
This PR is addressing issue #1767 but also fixes some typos in the user manual

##### What are the significant changes in functionality due to this change request?

----------------
For Change Control Board: Change Request Review
----------------
The following review must be completed by an authorized member of the Change Control Board.
- [x] 1. Review all computer code.
- [x] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual and xsd schema. If the input syntax change deprecates existing input files, a conversion script needs to be added (see Conversion Scripts).
- [x] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://github.com/idaholab/raven/wiki/RAVEN-Code-Standards#python) for details.
- [x] 4. Automated Tests should pass, including run_tests, pylint, manual building and xsd tests. If there are changes to Simulation.py or JobHandler.py the qsub tests must pass.
- [x] 5. If significant functionality is added, there must  be tests added to check this. Tests should cover all possible options.  Multiple short tests are preferred over one large test. If new development on the internal JobHandler parallel system is performed, a cluster test must be added setting, in <RunInfo> XML block, the node ```<internalParallel>``` to True.
- [x] 6. If the change modifies or adds a requirement or a requirement based test case, the Change Control Board's Chair or designee also needs to approve the change.  The requirements and the requirements test shall be in sync.
- [x] 7. The merge request must reference an issue.  If the issue is closed, the issue close checklist shall be done.
- [x] 8. If an analytic test is changed/added is the the analytic documentation updated/added?
- [x] 9. If any test used as a basis for documentation examples (currently found in `raven/tests/framework/user_guide` and `raven/docs/workshop`) have been changed, the associated documentation must be reviewed and assured the text matches the example.

### CODE REVIEW

File: doc/user_manual/code_interfaces/dymola.tex [hunk 1]
```diff
@@ -50,7 +50,7 @@ \subsubsection{Files}
...
+The Dymola interface can only pass scalar values into the TEXT parameter file. If the user wants to pass vector information to Dymola, he can do so by providing an optional TEXT vector file to Dymola. This file must have the type 'DymolaVectors'. This additional file can then be read by the Dymola model. If vector data is passed from RAVEN to the Dymola interface and the TEXT vector file is not specified, the interface will display an error and stop the Dymola execution. If the TEXT vector file is specified (and vector data is passed to the interface), the interface will write the data into the specified file, but also display a warning, saying that the Dymola interface found vector data to be passed and if this data is supposed to go into the simulation initialization file of type 'DymolaInitialisation' the array must be split into scalars. The \textless Files\textgreater specification for the vector data look as follows:
```
Comment (wangcj05) [inline doc/user_manual/code_interfaces/dymola.tex:53]: not related to this PR, may be worth to change all '' to `' or change "" to ``''

File: doc/user_manual/couplingAcode.tex [hunk 2]
```diff
@@ -413,6 +413,26 @@ \subsubsection{Method: \texttt{setCsvLoadUtil}}
...
+file:///Users/mandd/projects/raven/doc/user_manual/code_interfaces/phisics.tex
+file:///Users/mandd/projects/raven/doc/user_manual/code_interfaces/phisicsRelap5.tex
+file:///Users/mandd/projects/raven/doc/user_manual/code_interfaces/prescient.tex
+file:///Users/mandd/projects/raven/doc/user_manual/code_interfaces/rattlesnake.tex
+file:///Users/mandd/projects/raven/doc/user_manual/code_interfaces/ravenRunningRaven.tex
+file:///Users/mandd/projects/raven/doc/user_manual/code_interfaces/relap5.tex
+file:///Users/mandd/projects/raven/doc/user_manual/code_interfaces/relap7.tex
+file:///Users/mandd/projects/raven/doc/user_manual/code_interfaces/saphire.tex
+file:///Users/mandd/projects/raven/doc/user_manual/code_interfaces/scale.tex
+file:///Users/mandd/projects/raven/doc/user_manual/code_interfaces/serpent.tex
```
Comment (wangcj05) [inline doc/user_manual/couplingAcode.tex:434]: why this lines are needed? Also the path to the files are unique to your computer.

File: doc/user_manual/libraries.tex [hunk 3]
```diff
@@ -0,0 +1,26 @@
+\begin{itemize}
```
Comment (wangcj05) [inline doc/user_manual/libraries.tex:1]: Do we need to add this to the repo? I think we will automatically generate this file

**Review by wangcj05 — CHANGES_REQUESTED** [review]
@mandd I have minor comments for you to consider

**Comment by wangcj05** [comment]
@mandd In addition, I do not see the fix for issue #1767

File: doc/user_manual/code_interfaces/dymola.tex (hunk 1 above)
Comment (mandd) [reply doc/user_manual/code_interfaces/dymola.tex:53]: I was wondering if this '' convention was intentional

File: doc/user_manual/couplingAcode.tex (hunk 2 above)
Comment (mandd) [reply doc/user_manual/couplingAcode.tex]: I am not sure what happened here, this is definitely not intentional

File: doc/user_manual/libraries.tex (hunk 3 above)
Comment (mandd) [reply doc/user_manual/libraries.tex]: I have added it by mistake

**Comment by mandd** [comment]
@wangcj05 : after this morning meeting I happened to fix some typos in some of the src files

File: framework/Steps/Step.py [hunk 4]
```diff
@@ -103,7 +103,7 @@ class cls.
     inputSpecification.addParam("re-seeding", InputTypes.StringType, descr=r"""
               this optional
               attribute could be used to control the seeding of the random number generator (RNG).
-              If inputted, the RNG can be reseeded. The value of this attribute
+              If inputted, the RNG can be re-seeded. The value of this attribute
```
Comment (wangcj05) [inline framework/Steps/Step.py:106]: one of the unit test is failed due to the updated test, I think you need to regold that test

**Review by wangcj05 — APPROVED** [review]

**Comment by wangcj05** [comment]
@mandd Do you want me to merge this PR or do you still working on it?

**Comment by mandd** [comment]
> Details

This PR can be merged. TY

**Review by wangcj05 — APPROVED** [review]

**Pull request merged**